### DISCUSSION TITLE
Crop images

### DISCUSSION THREAD

**Discussion by gillez** [photoprism/photoprism#3785] [Ideas]
Hi,
I asked this in the general board but didn't get a response and thinking maybe that's the wrong board for it and "Ideas" is the correct place.

Is it possible to crop images in Photoprism, modifying the original or making a copy? If not, is there a plan to add this functionality?
I am moving to Photoprism from Google Photos after buying a Synology NAS. The one feature I really miss from Google Photos is the ability to crop photos. I don't think any other photo editing is necessary but crop would be really useful. I feel this should be core functionality.

Thanks,
Giles

**Comment by chrisbansart** [comment]
I've just posted a proposition on that here : https://github.com/photoprism/photoprism/discussions/5360
